### DISCUSSION TITLE
Mobile Discussion Editing Tag adding

### DISCUSSION THREAD

**Discussion by CuzImBisonratte** [community/community#39833] [Mobile] [answered]
When trying to add tags to a discussion, I get an error because I don't have the rights to edit Tags.  

I think the tag-editor-dialogue shouldn't be accessible if I don't have the rights to change anything in there.

https://user-images.githubusercontent.com/50182241/203541581-f1e4e3a7-9e7d-4a39-8705-fc0565d21c01.MOV

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.90.0 (84076705)
Device: iPad Air 5 (iOS 16.1)
</details>

**Comment by taki-on** [comment, marked as answer]
Thank you for reporting the bug! We will triage it soon and plan to fix it. Please stay tuned!
